src/app/components/book-dialog/add/add-dialog.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Inject, ChangeDetectionStrategy, EventEmitter } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { IReadBook } from '@app/shared/models/read-book.model';
+import { FormActions } from '@app/shared/enums';
+
+@Component({
+  selector: 'app-add-dialog',
+  templateUrl: './add-dialog.component.html',
+  styleUrls: ['../book-dialog.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AddDialogComponent {
+  private readonly _action = FormActions.Add;
+  private _newBook: IReadBook;
+  private _formInvalid: boolean;
+  private _dialogClosed: EventEmitter<boolean> = new EventEmitter();
+    
+  constructor(
+    private _dialogRef: MatDialogRef<AddDialogComponent>,
+    @Inject(MAT_DIALOG_DATA) readonly book: IReadBook
+    ) {}
+  
+  getNewBook($event: IReadBook) { 
+    this._newBook = $event;
+  }
+
+  getFormStatus($event: boolean) {
+    this._formInvalid = $event;
+  }
+
+  onSubmit() {
+    this._dialogClosed.emit(true);
+    this._dialogRef.close(this._newBook); 
+  }
+
+  onNoClick() {
+    this._dialogRef.close();
+  }
+
+  get action(): FormActions {
+    return this._action;
+  }
+
+	get formInvalid(): boolean {
```

Comment:
What's going on with alignment here?

Reply:
It's a Github issue. When you select changes from this commit, everything looks ok.
[https://github.com/agnieszka-pytel/angular-course-project/pull/5/commits/b4dd7e7b872003c17902cc0b1bf76b3553ac6ad3#diff-5c4fcda05ad5c13722e1c12913702d25](https://github.com/agnieszka-pytel/angular-course-project/pull/5/commits/b4dd7e7b872003c17902cc0b1bf76b3553ac6ad3#diff-5c4fcda05ad5c13722e1c12913702d25)